python/tests/backends/test_IonQ.py

```diff
@@ -0,0 +1,144 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+import cudaq, pytest, os, time
+from cudaq import spin
+from mock_ionq import startServer
+from multiprocessing import Process
+
+# Define the port for the mock server
+port = 62444
+
+
+def assert_close(want, got, tolerance=1.0e-5) -> bool:
+    return abs(want - got) < tolerance
+
+
+@pytest.fixture(scope="session", autouse=True)
+def startUpMockServer():
+    # We need a Fake Credentials Config file
```

Comment:
This setup doesn't seem to match the server helper. 
```suggestion
    # We need a Fake Credentials Config file
    os.environ["IONQ_API_KEY"] = "00000000000000000000000000000000"
```